### ISSUE TITLE
JavaScript: Menu not keyboard accessible.

### ISSUE THREAD

**Issue by yatil** [w3c/web-roadmaps#227]
When the menu is opened, the focus should be set to the first item of the menu. The focus should also wrap around (go from the last item to the first item on `tab` and from the first item to the last item when pressing `shift` + `tab`).

**Comment by tidoust** [comment]
@yatil, many thanks for taking the time to report UI issues, and many thanks for the very actionable issues you created! Most of them should now be fixed and I'm going to close the issues as a result. Feel free to re-open them if the fixes are not good enough or if they actually create other issues!

The menu should now be keyboard accessible:
- `Tab` (and `shift` + `Tab`), `Up`, `Down`, `Home`, `End`, `Pageup`, `Pagedown` can be used to navigate the menu.
- `Enter` triggers the link as usual
- `m` opens/closes the menu
- `Esc` closes the menu if it is opened

When the menu is closed, focus goes back to the last element that had focus before the menu was opened.

**Comment by yatil** [comment]
Thanks @tidoust!
